gradle/libs.versions.toml

```diff
@@ -3,10 +3,10 @@ androidx-activity = "1.10.1"
 androidx-benchmark = "1.3.4"
 androidx-lifecycle = "2.8.7"
 androidx-tracing-perfetto = "1.0.0"
-atomicfu = "0.26.1"
```

Comment:
Aligns our atomicfu version to `0.27.0`. Previously we were setting `0.26.1` in some places.